src/common/base/src/base/runtime_tracker.rs

```diff
@@ -23,38 +24,27 @@ use std::sync::Arc;
 use std::task::Context;
 use std::task::Poll;
 
-use futures::FutureExt;
+use pin_project_lite::pin_project;
+
+/// The root tracker.
+///
+/// Every alloc/dealloc stat will be fed to this tracker.
+static GLOBAL_TRACKER: MemoryTracker = MemoryTracker::empty();
```

Comment:
Also need global buffer.

Reply:
The buffer resides in every ThreadTracker.